projects/rocprofiler-systems/source/lib/core/trace_cache/storage_parser.hpp

```diff
@@ -64,12 +66,16 @@ class storage_parser
 
         if(_type_processing == nullptr)
         {
+            LOG_CRITICAL("TypeProcessing is nullptr");
             throw std::runtime_error("TypeProcessing is nullptr");
         }
 
+        LOG_DEBUG("Loading storage from file: {}", m_filename);
+
         std::ifstream ifs(m_filename, std::ios::binary);
         if(!ifs.good())
         {
+            LOG_ERROR("Failed to open file for reading: {}", m_filename);
             std::stringstream ss;
             ss << "Error opening file for reading: " << m_filename << "\n";
             throw std::runtime_error(ss.str());
```

Comment:
don't forget to remove sstream include header
```suggestion
            throw std::runtime_error(fmt::format("Error opening file for reading: {}", m_filename);
```